config/global.ini.php

```diff
@@ -566,6 +566,11 @@
 ; Recommended to keep enabled for best security.
 auth_token_rotation_notification_days = 180
 
+; Number of days before the expiration date of a personal auth token, where an email notification is sent to the user.
+; If set to -1 days, notifications won't be sent. 
+; Recommended to keep enabled for best security.
+auth_token_expiration_notification_days = 30
```

Comment:
To not forget it: changing the config file requires updating the `UIIntegrationTest_admin_diagnostics_configfile.png` screenshot :eyes:

Reply:
That is done in the back-end PR